pkg/build/api/validation/validation.go

```diff
@@ -76,6 +76,17 @@ func ValidateBuildConfig(config *buildapi.BuildConfig) fielderrors.ValidationErr
 	}
 
 	allErrs = append(allErrs, validateBuildSpec(&config.Spec.BuildSpec).Prefix("spec")...)
+
+	// validate ImageChangeTriggers of DockerStrategy builds
```

Comment:
validateTrigger validates ImageChange.From
validate[Source/Custom/Docker]Strategy validates the Strategy.From